main.py

```diff
@@ -140,6 +140,19 @@ def change_wallpaper(db, arg):
             print_columns(suggestions[1:])
             scripter.change_wallpaper(suggestions[0])
 
+def multiple_argument_handler(arg, time):
+    print("hi me", time)
+    db = Database()
+    if arg == "slideshow":
+        slideshow(db, 1, 494, time)
+    elif arg == "slideshow-kanto":
+        slideshow(db, 1, 152, time)
+    elif arg == "slideshow-johto":
+        slideshow(db, 152, 252, time)
+    elif arg == "slideshow-hoenn":
+        slideshow(db, 252, 387, time)
+    elif arg == "slideshow-sinnoh":
+        slideshow(db, 387, 494, time)
```

Comment:
What if the user inputs an invalid command or a typo:
```$ pokemon slideshow-canada 2```
There should be an else statement here to let the user know that their input is incorrect, instead of just remaining silent and doing nothing.